config/locales/en_etm.yml

```diff
@@ -331,6 +332,43 @@ en:
     feature_scenario: 'Feature this scenario'
     featured_scenario: 'Featured scenario'
     created_with: 'Scenario created with the Energy Transition Model'
+    users:
+      manage: 'Manage access'
+      manage_description:
+        Here you can manage who has access to your scenario and in which manner.
+        Click the 'Invite' button to give someone access to your scenario,
+        or use the table below to manage users that are already coupled.<br /><br />
+        Please visit the
+        <a href="https://docs.energytransitionmodel.com/main/#12-saved-scenario-access-management-page" target=\"_blank\">
+        documentation</a> for more information and details on user access management, such as user roles.
```

Comment:
Update this now that it is at the bottom of the page @mabijkerk @Charlottevm @kaskranenburgQ